src/ripple/app/tx/impl/details/NFTokenUtils.cpp

```diff
@@ -191,6 +191,7 @@ getPageForToken(
         : carr[0].getFieldH256(sfNFTokenID);
 
     auto np = std::make_shared<SLE>(keylet::nftpage(base, tokenIDForNewPage));
+    assert(np->key() > base.key);
```

Comment:
What's the purpose of this new assert?

Reply:
Er, I added that to make sure that I was understanding the key generation process correctly, and left it there because it didn't do any harm. It can be removed if you'd like.

Reply:
Done